framework/include/fviks/FVInterfaceKernel.h

```diff
@@ -123,6 +123,11 @@ class FVInterfaceKernel : public MooseObject,
    */
   const MooseVariableFV<Real> & var2() const { return _var2; }
 
+  /**
+   * @return The underlying assembly object, to work on residuals for example
+   */
+  Assembly & assembly() const { return _assembly; }
```

Comment:
There’s not much point to this. You’re returning a non const reference so this does not provide any increased encapsulation relative to having the data member protected.

Reply:
It’s also misleading to label the method as const when you’re returning non const data

Reply:
ok changed it.
I was doing this to avoid the re-ordering issue. I think the current re-ordering is ok?